lib/livebook/runtime/definitions.ex

```diff
@@ -427,9 +427,15 @@ defmodule Livebook.Runtime.Definitions do
 
       tabs =
         for sheet <- XlsxReader.sheet_names(package) do
-          # Assume the first row contains column names
-          {:ok, [header | rows]} = XlsxReader.sheet(package, sheet)
-          maps = Enum.map(rows, fn row -> header |> Enum.zip(row) |> Map.new() end)
+          maps =
+            case XlsxReader.sheet(package, sheet) do
```

Comment:
Do we also need to pass a flag for extract_chars? Also, should we bump the version to 0.8.4?

Reply:
expect_chars being returned was a bug after all. Will push an update to use new version.